### PR TITLE
adding welltestpy

### PR DESCRIPTION
**Pull request by MuellerSeb**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by MuellerSeb** [comment]
ahh... the `matplotlib-base` thing maybe.

**Comment by MuellerSeb** [comment]
@conda-forge-admin, please restart ci

File: recipes/welltestpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  source_files:
+    - tests
+  commands:
+    - pytest -vv tests
+
+about:
+  home: "https://github.com/GeoStat-Framework/welltestpy"
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (chrisburr) [inline recipes/welltestpy/meta.yaml:46]: A licence file is needed 
```suggestion
  license_file: LICENSE
```

File: recipes/welltestpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools >=45
+    - setuptools_scm >=3.4
+  run:
+    - python
```
Comment (chrisburr) [inline recipes/welltestpy/meta.yaml:24]: This needs to match
```suggestion
    - python >=3.5
```

**Comment by MuellerSeb** [comment]
@chrisburr thanks for merging. Why is the windows build still failing?

**Comment by chrisburr** [comment]
It doesn't matter as `noarch: python` packages are only built on Linux but the resulting binary can then be installed on any platform.

As for the actual cause, for some reason Python 2.7 finding its way into the build. I'm not sure why, it's possibly a bug in the staged-recipes CI as Python 2.7 support has been dropped from conda-forge. If it doesn't show up again it's not worth looking into.

**Pull request merged**